packages/plugin-block-charts/src/client/chart/antd/tools/renderText.ts

```
@@ -0,0 +1,23 @@
+interface IOptions {
+  fieldProps: { [key: string]: any };
+  dataIndex: string;
+  render: Function | undefined;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)